packages/block-library/src/cover/edit/index.js

```diff
@@ -211,23 +211,31 @@ function CoverEdit( {
 		overlayColor,
 	};
 
+	const coverBlockControls = (
```

Comment:
Minor nitpick (non-blocking): Should we drop the `cover` prefix for variables? This will match the naming convention used in other blocks.